script/workLoggingSystemFrames.js

```diff
@@ -49,24 +45,24 @@ frameDetails += idDetails;
 frameDetails += "\">";
 frameDetails += "<div class=\"col\">";
 frameDetails += "<div class=\"row part\">";
-frameDetails += "<div class=\"col-md-2\" id=\"logWorkButton\">";
+frameDetails += "<div class=\"col-md-2 logBtn\" id=\"logWorkButton\">";
```

Comment:
Please avoid using new class, try to use the id for css change

Reply:
I have used this because the id "logWorkButton" used at two places.

Reply:
Try different approach to solve this